lib/evmone/instructions_calls.cpp

```diff
@@ -182,6 +182,10 @@ Result create_impl(StackTop stack, int64_t gas_left, ExecutionState& state) noex
         // init_code_offset may be garbage if init_code_size == 0.
         msg.input_data = &state.memory[init_code_offset];
         msg.input_size = init_code_size;
+
+        // EOF initcode is not allowed for legacy creation
+        if (is_eof_container({msg.input_data, msg.input_size}))
```

Comment:
I think this unnecessarily modifies current behavior. If the initcode starts with `EF` it fails during execution.

Reply:
Do you mean to change the spec? I think motivation for light failure was better UX and being aligned with legacy creation transaction being invalid with EOF.
https://github.com/ipsilon/eof/pull/53
cc @pdobacz

Reply:
Yup, if we skip this, this is a change to the spec we produced (in both aspects as Andrei noted: CREATE and create tx, as they should remain aligned). I don't have a preference except that we settle the spec soon.